app/controllers/concerns/high_voltage/static_page.rb

```diff
@@ -22,14 +22,18 @@ module HighVoltage::StaticPage
     if respond_to?(:caches_page)
       caches_page :show, if: -> { HighVoltage.page_caching }
     end
-
-    hide_action :current_page, :page_finder, :page_finder_factory
```

Comment:
Since we'll be supporting older versions of Rails we should probably move this to a conditional in the same vein as we did for `caches_page` and `caches_action`